rsconnect/metadata.py

```diff
@@ -18,6 +18,7 @@
 from .log import logger
 from .models import AppMode, AppModes
 
+METADATA_RECORD_VERSION = 1
```

Comment:
Metadata comprises DataStore, ServerData, AppStore. Unless we are sure all three evolve at the same time and at the same rate, this would probably better off implemented as an appstore_version first.